src/mongo/MongoScrapbook.ts

```diff
@@ -105,14 +106,30 @@ async function executeCommand(context: IActionContext, command: MongoCommand): P
 
         // we don't handle chained commands so we can only handle "find" if isn't chained
         if (command.name === 'find' && !command.chained) {
-            await ext.editorManager.showDocument(context, new MongoFindResultEditor(database, command), 'cosmos-result.json', { showInNextColumn: true });
+            const db = await database.connectToDb();
+            const collectionName: string = nonNullProp(command, 'collection');
+            const collection: Collection = db.collection(collectionName);
+            // NOTE: Intentionally creating a _new_ tree item rather than searching for a cached node in the tree because
+            // the executed 'find' command could have a filter or projection that is not handled by a cached tree node
```

Comment:
Would it be possible to update the existing collection tree item instead of creating a new one? Or is this easier?

Reply:
We _could_ update the existing collection tree item, but we don't _want_ to. The find command could have a filter or a projection in it's arguments, meaning the resulting documents could be completely different than what's in the tree. For example if the collection tree item has these documents:
```json
[
  {
    "_id": "apples",
    "qty": 5
  },
  {
    "_id": "avocados",
    "qty": "fourteen"
  },
  {
    "_id": "bananas",
    "qty": 26
  },
  {
    "_id": "oranges",
    "qty": {
      "in stock": 8,
      "ordered": 12
    }
  }
]
```

I can run a find command like this:
```
db.fruit.find( { qty: { $gt: 4 } } )
```

resulting in this:
```json
[
  {
    "_id": "apples",
    "qty": 5
  },
  {
    "_id": "bananas",
    "qty": 26
  }
]
```

We want the tree to still show all 4 documents, not just those 2. This piece is actually very similar to how it was before: https://github.com/microsoft/vscode-cosmosdb/pull/1572/files#diff-0cdcd82c7e289824f6c83cc3f04581d2L37